docs/gameobject.md

```diff
@@ -95,28 +102,32 @@ This is a bitmask field that describes all the phases that this gameobject will
 
 ### position\_x
 
-The X position.
-
 ### position\_y
 
-The Y position.
-
 ### position\_z
```

Comment:
I'd keep one description per axis.

i.e which axis is X, which is Y which is Z